applications/tari_console_wallet/src/main.rs

```diff
@@ -77,23 +77,7 @@ fn main_inner() -> Result<(), ExitCodes> {
     // check for recovery based on existence of wallet file
     let mut boot_mode = boot(&bootstrap, &config)?;
 
-    let master_key: Option<PrivateKey> = if matches!(boot_mode, WalletBoot::Recovery) {
-        let private_key = if bootstrap.seed_words.is_some() {
-            let seed_words: Vec<String> = bootstrap
-                .seed_words
-                .clone()
-                .unwrap()
-                .split_whitespace()
-                .map(|v| v.to_string())
-                .collect();
-            get_private_key_from_seed_words(seed_words)?
-        } else {
-            prompt_private_key_from_seed_words()?
-        };
-        Some(private_key)
-    } else {
-        None
-    };
+    let master_key: Option<PrivateKey> = get_master_key(boot_mode, &bootstrap)?;
```

Comment:
Is the return type necessary in this declaration? I would have thought it could be inferred from the fn

Reply:
I tried to keep the code the same, just moving it around